detectron2/modeling/meta_arch/retinanet.py

```diff
@@ -92,12 +93,53 @@ def __init__(self, cfg):
             cfg.MODEL.RETINANET.IOU_LABELS,
             allow_low_quality_matches=True,
         )
+        self.vis_period = cfg.VIS_PERIOD
+        self.input_format = cfg.INPUT.FORMAT
 
         pixel_mean = torch.Tensor(cfg.MODEL.PIXEL_MEAN).to(self.device).view(3, 1, 1)
         pixel_std = torch.Tensor(cfg.MODEL.PIXEL_STD).to(self.device).view(3, 1, 1)
         self.normalizer = lambda x: (x - pixel_mean) / pixel_std
         self.to(self.device)
 
+    def visualize_training(self, batched_inputs, results):
+        """
+        A function used to visualize images and intermidiate network results. It shows ground truth
```

Comment:
not "intermediate" -- it visualizes the final predictions

Reply:
Fixed in 45d9600